### PR TITLE
Add example show how to configure proxy for kube

### PR DESCRIPTION
**Pull request by netroby**
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2639)

### CODE REVIEW

**Comment by chenopis** [comment]
@netroby If .vscode/temp.sql is just a local temp file, you should probably remove it from the pull request. Thanks for the contribution! =)

**Comment by netroby** [comment]
Done, sir @chenopis

**Comment by chenopis** [comment]
@netroby Great, thanks!

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -260,6 +260,22 @@ These environment variables are a short-term solution, eventually they will be i
 
 If you want to use kubeadm with an http proxy, you may need to configure it to support http_proxy, https_proxy, or no_proxy.
 
+For example, your kube master node ip is 10.18.17.16, and you have a proxy support both http/https on 10.18.17.16 port 8080
```
Comment (chenopis) [inline docs/admin/kubeadm.md]: ### Revision suggestion:

For example, if your kube master node IP address is 10.18.17.16 and you have proxy support both http/https on 10.18.17.16 port 8080, you can use the following command:

File: docs/admin/kubeadm.md [hunk 2]
```diff
@@ -260,6 +260,22 @@ These environment variables are a short-term solution, eventually they will be i
...
+export PROXY_PORT=8080
+export PROXY_IP=10.18.17.16
+export http_proxy=http://$PROXY_IP:$PROXY_PORT
+export HTTP_PROXY=$http_proxy
+export https_proxy=$http_proxy
+export HTTPS_PROXY=$http_proxy
+export no_proxy="localhost,127.0.0.1,localaddress,.localdomain.com,example.com,10.18.17.16"
+```
+
+Remember change ```proxy_ip``` and add kube master node ip to ```no_proxy```
```
Comment (chenopis) [inline docs/admin/kubeadm.md]: ### Revision suggestion:

Remember to change \```proxy_ip\``` and add a kube master node IP address to \```no_proxy\```.

**Comment by netroby** [comment]
@chenopis Thanks for suggestion, revision as you describe

**Comment by chenopis** [comment]
Great, thank you @netroby ! Looking forward to more of your contributions. =)

**Pull request merged**